api/backup.go

```diff
@@ -37,6 +37,8 @@ type BackupType string
 const (
 	// BackupTypeAutomatic is the type for automatic backup.
 	BackupTypeAutomatic BackupType = "AUTOMATIC"
+	// BackupTypeAutomaticPITRCutover is the type of backup taken at PITR cutover stage.
+	BackupTypeAutomaticPITRCutover BackupType = "AUTOMATIC_PITR_CUTOVER"
```

Comment:
Not sure if that's a good name. Simply PITR?

Reply:
SG.